src-ThirdParty/NUnitLite/Constraints/XmlSerializableConstraint.cs

```diff
@@ -58,7 +59,8 @@ public override bool Matches(object actual)
 
                 stream.Seek(0, SeekOrigin.Begin);
 
-                object value = serializer.Deserialize(stream);
+                var reader = XmlReader.Create (stream, new XmlReaderSettings { XmlResolver = null });
+                object value = serializer.Deserialize(reader);
```

Comment:
I think `XmlReader` needs to be in a `using` block.

Reply:
Good call, updated these 3 areas.